README.md

```diff
@@ -1170,6 +1170,8 @@ Objective-C:
 
 ### Get a structure data
 
+*Please note this API has been deprecated in version 7.0.0 of the iOS SDK. If you require this API please contact [support@thunderhead.com](mailto:support@thunderhead.com).*
```

Comment:
Not sure why it is in red https://github.com/thunderheadone/one-sdk-ios/blob/5b8c109cc72e23fe57b9438eb30d2aedd59d57dd/README.md#get-a-structure-data